docs/key-issues/gpus.md

```diff
@@ -30,54 +30,78 @@ Packaging such projects for PyPI has been, and still is, quite challenging.
+other CUDA versions and a CPU-only version in a separate wheelhouse (see
 [pytorch.org/get-started](https://pytorch.org/get-started/locally/)).
 CuPy provides a number of packages: [`cupy`](https://pypi.org/project/cupy/),
 [`cupy-cuda102`](https://pypi.org/project/cupy-cuda102/),
 [`cupy-cuda110`](https://pypi.org/project/cupy-cuda1110/),
 [`cupy-cuda111`](https://pypi.org/project/cupy-cuda111/),
-[`cupy-cuda11x`](https://pypi.org/project/cupy-cuda11x/),
 [`cupy-rocm-4-3`](https://pypi.org/project/cupy-rocm-4-3/),
 [`cupy-rocm-5-0`](https://pypi.org/project/cupy-rocm-5-0/).
-Other projects do similar things - none of it works very well.
-
-GPU packages tend to result in very large wheels. This is true in particular
-for deep learning packages, because they link in
-[cuDNN](https://developer.nvidia.com/cudnn). For example, the most recent
-`manylinux2014` wheels for TensorFlow are 588 MB
-([2.11.0 files](https://pypi.org/project/tensorflow/2.11.0/#files)), and for
-PyTorch those are 890 MB ([1.13.0
-files](https://pypi.org/project/tensorflow/2.11.0/#files)). The problems around
-and causes of GPU wheel sizes were discussed in depth in
-[this Packaging thread on Discourse](https://discuss.python.org/t/what-to-do-about-gpus-and-the-built-distributions-that-support-them/7125).
+This works, but adds maintenance overhead to project developers and consumes
+more storage and network bandwidth for PyPI.org. Moreover, it also prevents
+downstream projects from properly declaring the dependency unless they also
+follow a similar multi-package approach. As of CUDA 11, CUDA promises
+[minor version
+compatibility](https://docs.nvidia.com/deploy/cuda-compatibility/index.html#minor-version-compatibility),
+which allows building packages compatible across an entire CUDA major
+version. CuPy now leverages this to produce wheels like 
+[`cupy-cuda11x`](https://pypi.org/project/cupy-cuda11x/) and
+[`cupy-cuda12x`](https://pypi.org/project/cupy-cuda12x/) that work for any CUDA
+11.x or CUDA 12.x version, respectively, that a user has installed. Libraries
+that package PTX code cannot take advantage of this process yet, however ([see
+below](#additional-notes-on-cuda-compatibility)).
+
+GPU packages tend to result in very large wheels. This is mainly because
+compiled GPU libraries must support a number of architectures, leading to large
+binary sizes. These effects are compounded by the requirements imposed by the
+manylinux standard for Linux wheels, which results in many large libraries being
+bundled into a single wheel (see [Native dependencies](native-dependencies) for
+details). This is true in particular for deep learning packages because they
+link in [cuDNN](https://developer.nvidia.com/cudnn). For example, recent
+`manylinux2014` wheels for TensorFlow are 588 MB ([2.11.0
+files](https://pypi.org/project/tensorflow/2.11.0/#files)), and for PyTorch
+those are 890 MB ([1.13.0
+files](https://pypi.org/project/torch/1.13.0/#files)). The problems around and
```

Comment:
Can we call the links here "wheels for v2.11.0" and "wheels for v1.13.0"? I find "2.11.0 files" unintuitive.

Reply:
Agreed, changing.